src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/OffenderCourseAttendanceBuilder.kt

```diff
@@ -38,31 +32,28 @@ class OffenderCourseAttendanceBuilderFactory(
 class OffenderCourseAttendanceBuilder(
   val repository: Repository,
   val eventId: Long,
-  val eventDate: LocalDate,
-  val startTime: LocalDateTime,
-  val endTime: LocalDateTime,
+  val courseSchedule: CourseSchedule,
   val eventStatusCode: String,
   val toInternalLocationId: Long?,
   val outcomeReasonCode: String?,
   val paidTransactionId: Long?,
-) {
+) : CourseAttendanceDsl {
   fun build(
-    courseSchedule: CourseSchedule,
     offenderProgramProfile: OffenderProgramProfile,
   ): OffenderCourseAttendance =
     OffenderCourseAttendance(
       eventId = eventId,
       offenderBooking = offenderProgramProfile.offenderBooking,
-      eventDate = eventDate,
-      startTime = startTime,
-      endTime = endTime,
+      eventDate = courseSchedule.scheduleDate,
+      startTime = courseSchedule.startTime,
+      endTime = courseSchedule.endTime,
```

Comment:
Now that we need to pass the course schedule into the attendance builder we might as well just default the course time from it.